src/filesystem.ts

```diff
@@ -138,3 +182,17 @@ async function addSampleData(fs: FileSystem): Promise<void> {
   await fs.mkdir({ directory: [ Branch.Private, "Video" ] })
   await fs.publish()
 }
+
+
+
+// 🔑
+
+
+export async function readKey(): Promise<string> {
```

Comment:
Probably could use some documentation to indicate that this read key is a top-level read key used with auth. Webnative also stores other read keys in the key store which are tied to specific private paths.

Reply:
Would this work be accurate?

https://github.com/fission-suite/webnative/blob/514a380480eaa599f88f986a901769a1c5aa94b4/src/filesystem.ts#L192-L195

My understanding is the read key can be used outside of an auth scenario to read the private filesystem with root access. Is that correct? If not, I may need to add more read keys for this PR to work.

Reply:
That's accurate and correct 👍 Other read keys are only received when coming back from the lobby with certain authorised permissions.